src/main/groovy/com/blogspot/toomuchcoding/testprofiler/TestProfilerPluginExtension.groovy

```diff
@@ -79,8 +79,20 @@ class TestProfilerPluginExtension {
         /**
          * A method to add additional suffixes
          */
-        void addTestClassNameSuffix(String testClassNameSuffix) {
-            testClassNameSuffixes << testClassNameSuffix
+        void addTestClassNameSuffix(String... testClassNameSuffix) {
```

Comment:
`...Suffixes`? You could add a new method and deprecate that to keep backward compatibility.